tests/unit/test_processors/test_otlp_metrics_processor.py

```diff
@@ -43,73 +43,73 @@ def test__init(self, mocker):
-        )
-        assert "foo-lambda-ffoooofofooooooofooofooooofofofofoooooofoooooooooffoffooooooffffofooooofffooooooofoooooffoofofoooooofffofooofoffoooofooofoooooooooooooofooffoooofofooofoooofoofooffooooofoofooooofoooooffoofffoffoooooofoooofoooffooffooofofooooooffffooofoooooofooo" == ResponseTimeProcessor(
-            mock_apm_config,
-        ).calculate_otlp_transaction_name("foo-span")
-
-    def test_calculate_otlp_transaction_name_env_lambda_truncated(self, mocker):
-        mock_apm_config = self.get_mock_apm_config(
-            mocker,
-            outer_txn_retval=None,
-            lambda_function_name="foo-lambda-name",
-        )
-        assert "foo-lambda-name" == ResponseTimeProcessor(
-            mock_apm_config,
-        ).calculate_otlp_transaction_name("foo-span")
-
-    def test_calculate_otlp_transaction_name_span_name(self, mocker):
-        mock_apm_config = self.get_mock_apm_config(
-            mocker,
-            outer_txn_retval=None,
-            lambda_function_name=None,
-        )
-        assert "foo-span" == ResponseTimeProcessor(
-            mock_apm_config,
-        ).calculate_otlp_transaction_name("foo-span")
-
-    def test_calculate_otlp_transaction_name_span_name_truncated(self, mocker):
-        mock_apm_config = self.get_mock_apm_config(
-            mocker,
-            outer_txn_retval=None,
-            lambda_function_name=None,
-        )
-        assert "foo-span-ffoooofofooooooofooofooooofofofofoooooofoooooooooffoffooooooffffofooooofffooooooofoooooffoofofoooooofffofooofoffoooofooofoooooooooooooofooffoooofofooofoooofoofooffooooofoofooooofoooooffoofffoffoooooofoooofoooffooffooofofooooooffffooofoooooofooooo" == ResponseTimeProcessor(
-            mock_apm_config,
-        ).calculate_otlp_transaction_name("foo-span-ffoooofofooooooofooofooooofofofofoooooofoooooooooffoffooooooffffofooooofffooooooofoooooffoofofoooooofffofooofoffoooofooofoooooooooooooofooffoooofofooofoooofoofooffooooofoofooooofoooooffoofffoffoooooofoooofoooffooffooofofooooooffffooofoooooofoooooofooofoooofoo")
-
-    def test_calculate_otlp_transaction_name_empty(self, mocker):
-        mock_apm_config = self.get_mock_apm_config(
-            mocker,
-            outer_txn_retval=None,
-            lambda_function_name=None,
-        )
-        assert "unknown" == ResponseTimeProcessor(
-            mock_apm_config,
-        ).calculate_otlp_transaction_name("")
+    # def test_calculate_otlp_transaction_name_env_trans(self, mocker):
+    #     mock_apm_config = self.get_mock_apm_config(
+    #         mocker,
+    #         "foo-env-trans-name",
+    #     )
+    #     assert "foo-env-trans-name" == ResponseTimeProcessor(
+    #         mock_apm_config,
+    #     ).calculate_otlp_transaction_name("foo-span")
+    #
+    # def test_calculate_otlp_transaction_name_env_trans_truncated(self, mocker):
```

Comment:
Oh right these. We're not truncating in ServiceEntryProcessor as we chatted offline yesterday, so we could completely remove these test cases maybe?

Reply:
Removed. Done